COPYING

```diff
@@ -70,7 +70,7 @@ OF THIS SOFTWARE OR ITS FITNESS FOR ANY PARTICULAR PURPOSE.
 
 
 
-jq uses parts of the open source C library "decNumber", which is distribured
+jq uses parts of the open source C library "decNumber", which is distributed
```

Comment:
A lot of the changes below are to this library... If it's still maintained in an upstream, I can send fixes to it...